packages/transaction-builder/src/index.ts

```diff
@@ -1 +1,204 @@
-export {}
+import { Buffer } from 'buffer'
+import {
+  MsgProtoAppStake,
+  MsgProtoAppUnstake,
+  MsgProtoNodeStakeTx,
+  MsgProtoNodeUnjail,
+  MsgProtoNodeUnstake,
+  MsgProtoSend,
+} from './models/msgs'
+import { AbstractProvider } from '@pokt-foundation/pocketjs-abstract-provider'
+import { KeyManager } from '@pokt-foundation/pocketjs-signer'
+import {
+  RawTxRequest,
+  TransactionResponse,
+} from '@pokt-foundation/pocketjs-types'
+import { TxEncoderFactory } from './factory/tx-encoder-factory'
+import { TxMsg, CoinDenom, TxSignature } from './models/'
+
+export type ChainID = 'mainnet' | 'testnet'
+
+export const BASE_FEE = '10000'
```

Comment:
Replace `BASE_FEE` with `BASE_FEE_UPOKT` anywhere else if needed

Reply:
At the chain level (tendermint) it doesn't understand decimals, so that's why we have the `uPOKT` denomination, similar to how ethereum has `wei` (`10^18`). It's better not to use them, even if functionality has been implemented to parse these. It's also why I'm using strings—tendermint only understands bignumbers and other internal fixed-point math. So TLDR: Won't be adding this 😄

Reply:
> At the chain level (tendermint) it doesn't understand decimals, so that's why we have the `uPOKT` denomination, similar to how ethereum has `wei` (`10^18`). It's better not to use them, even if functionality has been implemented to parse these. It's also why I'm using strings—tendermint only understands bignumbers and other internal fixed-point math. So TLDR: Won't be adding this smile

I think it would be worth using the constant value in the comparison to avoid overpaying the fee. I kinda overlooked that this variable is being used as default so it messes with the tendermint thing that you pointed out.

Reply:
Yeah, probably should've named it `DEFAULT_BASE_FEE`.